docs/DeveloperGuide.md

```diff
@@ -145,6 +146,8 @@ The `Storage` component,
 * inherits from both `AddressBookStorage` and `UserPrefStorage`, which means it can be treated as either one (if only the functionality of only one is needed).
 * depends on some classes in the `Model` component (because the `Storage` component's job is to save/retrieve objects that belong to the `Model`)
 
+Within the `/storage` folder, there also exists a `CsvManager` class that is a helper file to do file IO related operations when Import CSV or Export CSV is called. This file does not fit within the storage component diagram as it is simply meant to separate IO operations to a helper class within `/storage`.
```

Comment:
Likewise, please call it a package.
```suggestion
Within the `storage` package, there also exists a `CsvManager` class that is a helper that provides file IO related operations for working with CSV files. It is used in the Import CSV or Export CSV features. This class does not fit within the storage component diagram as it does not interact with any other classes, serving solely as a helper class.
```

Reply:
thanks for the suggestion, implemented in https://github.com/AY2122S2-CS2103-W17-1/tp/pull/203/commits/3193b02116aeddfa74a438d365ea5575d9acfb0b

Reply:
I think you might have missed this.

Reply:
oops yes i missed this, thanks! resolved in https://github.com/AY2122S2-CS2103-W17-1/tp/pull/203/commits/805457bc58e7b818e04fe9ea0926b906040f4fd5